common_utils/utils.py

```diff
@@ -1,8 +1,34 @@
 import json
+import logging
+import os
+import sys
 from pathlib import PurePath
 
 from crum import get_current_request
 from django.conf import settings
+from django.db import transaction
+from django.utils.translation import override
+from django_ilmoitin.models import NotificationTemplate
+from jinja2 import Environment, FileSystemLoader
+from parler.utils.context import switch_language
+
+from youths.enums import NotificationType as YouthNotificationType
+
+logger = logging.getLogger(__name__)
+handler = logging.StreamHandler()
+logger.setLevel(logging.DEBUG)
+# create console handler and set level to debug
+handler.setLevel(logging.DEBUG)
+
+# add ch to logger
+logger.addHandler(handler)
```

Comment:
Is there a better way to configure logging? This doesn't look correct to me.

Reply:
There is a better way. You should do the in the settings.py (there's the LOGGING setting).

In this particular case I would add the following 

```python
LOGGING = {
    ...
    "loggers": {
        ...
        "common_utils.utils": {
            "handlers": ["console"],
            "level": "INFO",
        },
    },
}
```